hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationServiceV1.java

```diff
@@ -110,6 +111,30 @@
         registerJavaTypeSerializers();
     }
 
+    @Override
+    public <B extends Data> B toData(Object obj, DataType type) {
```

Comment:
I am not very excited about fiddling with various data types in a regular serialization service.